api/v1alpha1/multiclusterservice_types.go

```diff
@@ -92,6 +93,13 @@ type ServiceSpec struct {
 	StopOnConflict bool `json:"stopOnConflict,omitempty"`
 	// Reload instances via rolling upgrade when a ConfigMap/Secret mounted as volume is modified.
 	Reload bool `json:"reload,omitempty"`
+
+	// +kubebuilder:default:=Continuous
+	// +kubebuilder:validation:Enum:=OneTime;Continuous;ContinuousWithDriftDetection;DryRun
+
+	SyncMode        string                            `json:"syncMode,omitempty"`
+	IgnoreDrift     []libsveltosv1beta1.PatchSelector `json:"ignoreDrift,omitempty"`
```

Comment:
It's a small thing but should we rename this to `DriftIgnore` to make it similar to `DriftExclusions`? 🤔

Reply:
makes sense to me